aggregate/PUMS/aggregate_PUMS.py

```diff
@@ -236,10 +237,22 @@ def add_category(self, indicator):
 
     def order_columns(self):
         """This can be DRY'd out, written quickly to meet deadline"""
-
-        col_order = []
-        for ind_denom in self.indicators_denom:
-            ind = ind_denom[0]
+        inds_to_order = [
+            ind_denom[0]
+            for ind_denom in self.indicators_denom
+            if ind_denom[0] != "total_pop"
+        ]
+        # Don't love hardcoding the beginning of this list, can be refactored
+        col_order = [
+            "total_pop-count",
+            "total_pop-count-cv",
+            "total_pop-count-moe",
+            "total_pop-pct",
+            "total_pop-pct-cv",
+            "total_pop-pct-denom",
```

Comment:
quick reference here to [one of Amanda's comment](https://github.com/NYCPlanning/db-equitable-development-tool/issues/80#issuecomment-1050981122) about leaving out the `total_pop-pct-denom` @SashaWeinstein